package.json

```diff
@@ -54,6 +56,33 @@
     ]
   },
   "devDependencies": {
+    "@babel/core": "^7.13.16",
+    "@babel/preset-env": "^7.13.15",
+    "@babel/preset-react": "^7.13.13",
+    "@testing-library/jest-dom": "^5.12.0",
+    "babel-jest": "^26.6.3",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.6",
+    "jest": "^26.6.0",
+    "jest-fetch-mock": "^3.0.3",
+    "jest-svg-transformer": "^1.0.0",
+    "jest-transform-css": "^2.1.0",
+    "react-test-renderer": "^17.0.2",
```

Comment:
Some of these devDependencies are already installed via react-scripts with specific compatible versions. I'd prefer to follow react-scripts' recommendation to not override any of its dependencies.

We can drop `@babel/core`, `@babel/present-env`, `@babel/preset-react`, `babel-jest`, and `jest`.

Also, the version of `react-test-renderer` should be `^16.14.0` since we're not using React 17.